lib/glific/providers/maytapi/message.ex

```diff
@@ -3,21 +3,45 @@ defmodule Glific.Providers.Maytapi.Message do
   Message API layer between application and Maytapi
   """
 
-  alias Glific.Providers.Maytapi.ApiClient
+  import Ecto.Query, warn: false
+
+  alias Glific.Partners
+
+  alias Glific.{
+    Messages,
+    Providers.Maytapi.ApiClient,
+    Repo,
+    WAGroup.WAManagedPhone
+  }
 
   @doc false
   @spec send_text(non_neg_integer, map()) :: any()
   def send_text(org_id, attrs) do
-    send_message(org_id, attrs)
-  end
+    phone_id = get_phone_id(attrs)
```

Comment:
If phone doesn't exist, then we will raise a runtime error here. Should we handle much more gracefully since this is a user facing api?

Reply:
but only glific staff will send the message?